### PR TITLE
david lus onboarding

### PR DESCRIPTION
**Pull request by d8lu**
# Strategy Onboarding

## TODO List

Onboarding tasks:

- [x] Task 1: Implement `fibonacci_term()` in `fibonacci.py`
  - All tests in `test_fibonacci.py` should pass, run `pytest` command to check
- [x] Task 2: Implement `find_min_index()` and `reverse_str_arr()`
  - Write tests that cover the logic that you wrote in `test_array.py`
- [x] Task 3: Process and visualize sample data in `matplot.py`
  - Read data in CSV using `pandas` or `csv`, handle the missing values, plot using `matplotlib`
  - Include a screenshot of your graph in the PR

Some extra items:
[](url)
- [ ] Completed [strategy welcome survey](https://forms.gle/SKozF7TUn8o385zd8)

### CODE REVIEW

**Comment by d8lu** [comment]
@rodrigotiscareno @probro27

File: onboarding/tests/test_array.py [hunk 1]
```diff
@@ -1 +1,9 @@
-# write some tests here!
+import pytest
+from ..array_1 import find_min_index
```
Comment (probro27) [inline onboarding/tests/test_array.py:2]: This should be array not array_1. That's causing the tests to fail. Before submitting check with `pytest` if your code is working fine :)

**Review by probro27 — CHANGES_REQUESTED** [review]
@d8lu Everything looks fine. Just make the change suggested to get the `pytest` to run successfully. 
Thanks

**Comment by probro27** [comment]
Hey @d8lu any updates on the PR?

**Pull request closed without merging**